src/main/target/NAZE/config.c

```diff
@@ -104,6 +106,11 @@ void targetConfiguration(void)
         compassConfigMutable()->interruptTag = IO_TAG(PB12);
     }
 #endif
+
+#ifdef BLACKBOX
+    if (hardwareRevision >= NAZE32_REV5)
```

Comment:
I think this will wedge NAZE32_SP devices on configuration reset, since they have the MPU on the SPI bus that has the flash chip on REV6 / REV6:

https://github.com/McGiverGim/cleanflight/blob/c16d050e2be2cd2d4738b22249a0c919964f48b9/src/main/target/NAZE/hardware_revision.c#L100

@hydra: Do you happen to have a NAZE32_SP board that you could test this with?